ludwig/api.py

```diff
@@ -111,7 +111,7 @@
 
 @PublicAPI
 @dataclass
-class EvaluationFrequency:
+class EvaluationFrequency:  # noqa F821
```

Comment:
Hmm, do you know why the linter complains about this? I don't see anything immediately wrong with the way this class is defined.

Reply:
@tgaddair I see the complaint in PyCharm -- they only reason I could think of is that the dataclass we use is not from Python but from Marshmallow, and perhaps PyCharm is unable to introspect it in sufficient depth.

Reply:
Ah, I see. Thanks for the clarification. Sounds like this might require more digging at some point.

cc @ksbrar

Reply:
Thanks for flagging! I have made a note to check this out later.